src/main/java/controllers/AddPersonalServlet.java

```diff
@@ -0,0 +1,113 @@
+package controllers;
+
+import dao.DaoFactory;
+import dao.PersonalDao;
+import model.Personal;
+import model.Role;
+import utils.HashGenerator;
+import utils.StringFieldValidate;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.util.List;
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+/*
+сервлет для добавления персонала
+ */
+@WebServlet("/addPersonal")
+public class AddPersonalServlet extends HttpServlet {
+    DaoFactory daoFactory;
+    HashGenerator hashGenerator;
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException,
+            IOException {
+//        super.doGet(request, response);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        HttpSession session = request.getSession(true);
+        Locale locale = (Locale) session.getAttribute("locale");
+        if (locale == null) {
+            locale = new Locale("ru");
+        }
+        ResourceBundle bundle = ResourceBundle.getBundle("message", locale);
+        response.setContentType("text/html;charset=utf-8");
+        String firstName = request.getParameter("firstName").trim();
+        firstName = new String(firstName.getBytes("ISO-8859-1"), "UTF-8");
+        String lastName = request.getParameter("lastName").trim();
+        lastName = new String(lastName.getBytes("ISO-8859-1"), "UTF-8");
+        String login = request.getParameter("login").trim();
+        String password = request.getParameter("password").trim();
+        String role = request.getParameter("role");
+        StringFieldValidate stringFieldValidate = new StringFieldValidate();
+        boolean isValid = stringFieldValidate.doValidation(firstName);
+        if (isValid) {
```

Comment:
What if you have to check 20 fields? You will get nice stairs i think :)
So it should be in separate method with instance of Personal